.travis.yml

```diff
@@ -172,8 +172,11 @@ jobs:
       name: "Pushing release to hub.docker.com"
     - script:
       - bin/s3 sync s3://travis-$TRAVIS_BUILD_NUMBER/build-artifacts build/package
-      - bin/release_android "$TRAVIS_TAG" "$BINTRAY_USER" "$BINTRAY_TOKEN"
-      name: "Publish Android SDK to Bintray and JCenter"
+      - openssl aes-256-cbc -K $encrypted_2698a83519b3_key -iv $encrypted_2698a83519b3_iv -in bin/package/android/key.gpg.enc -out bin/package/android/key.gpg -d
```

Comment:
Security issue here. We should inject this key from environment variables

Reply:
This is the Travis way to provide secure keys.
https://docs.travis-ci.com/user/encrypting-files/
What is the security issue?